pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/dashboard/QueryBuildPayload.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.dsm.model.dashboard;
+
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.db.dto.dashboard.DashboardLabelDto;
+import org.broadinstitute.dsm.db.dto.ddp.instance.DDPInstanceDto;
+import org.broadinstitute.dsm.model.elastic.filter.AndOrFilterSeparator;
+import org.broadinstitute.dsm.model.elastic.filter.query.BaseQueryBuilder;
+
+@Getter
```

Comment:
I don't understand what this class is built for, can you add a comment explaining?